mzLib/Test/TestReaders/TestMgf.cs

```diff
@@ -55,19 +75,32 @@ public static void TestLoadMgf()
             Assert.AreEqual(1294963.5999999996, ya.TotalIonCurrent);
             Assert.AreEqual(110.0719, ya.ScanWindowRange.Minimum);
             Assert.AreEqual(1038.8018, ya.ScanWindowRange.Maximum);
-            var ya2 = a.GetOneBasedScan(20).MassSpectrum;
+            var ya2 = reader.GetOneBasedScan(20).MassSpectrum;
             Assert.AreEqual(165, ya2.Size);
-            var ya3 = a.GetOneBasedScan(2).MassSpectrum;
+            var ya3 = reader.GetOneBasedScan(2).MassSpectrum;
             Assert.AreEqual(551, ya3.Size);
         }
 
         [Test]
-        public static void TestLoadMgfTabSeparated()
+        public static void TestErrorMgfContainsDuplicateScanNumbers()
         {
+            string path = Path.Combine(TestContext.CurrentContext.TestDirectory, "DataFiles", "Tab_separated_peak_listWithDuplicateScan.mgf");
+            MsDataFile dataReader = ReaderCreator.CreateReader(path);
+            dataReader.LoadAllStaticData();
 
-            Mgf a = Mgf.LoadAllStaticData(Path.Combine(TestContext.CurrentContext.TestDirectory, "DataFiles", "Tab_separated_peak_list.mgf"));
+            //There are two scans with scan #3. This is not allowed and an MzLibException is thrown
+            Assert.Throws<MzLibException>(() => dataReader.InitiateDynamicConnection());
+        }
+
+        [Test]
+        public static void TestLoadMgfTabSeparated()
+        {
+            string path = Path.Combine(TestContext.CurrentContext.TestDirectory, "DataFiles", "Tab_separated_peak_list.mgf");
+            var dataReader = ReaderCreator.CreateReader(path); 
+            dataReader.LoadAllStaticData();
```

Comment:
It's apparently having trouble finding this file. Looks like it's named "Tab_separated_peak_list - Copy.mgf" in the file tree, which is probably the issue.